locales/en.json

```diff
@@ -49,8 +49,8 @@
 	"Help": "Help",
 	"Upload Image": "Upload Image",
 	"Menu": "Menu",
-	"This page need refresh": "This page need refresh",
-	"You have an incompatible client version.": "You have an incompatible client version.",
+	"This page need refresh": "This page needs a refresh",
```

Comment:
I can't get this one.
Can the "a" be ignored?
I would prefer shorter and clear wording in most of the UI text.

Reply:
According to a native English speaker: "This page needs to be refreshed" is preferred.